### ISSUE TITLE
How to specify 'spacy<3' in 'spacy_install(pip = TRUE)' under the conda environment?

### ISSUE THREAD

**Issue by paithiov909** [quanteda/spacyr#205]
I would like to install spacy of specific version using pip under the conda environment (even though I know that's not a good practice).

For instance, this snippet works fine as I expected.

```r
require(reticulate)
conda_create("spacy_condaenv", python_version = "3.9")
conda_install("spacy_condaenv", packages = "spacy<3", pip = TRUE)

require(spacyr)
spacy_download_langmodel("ja_core_news_sm")
```

I tried to do this just using `spacyr::spacy_install`, but this does not work.

```r
spacyr::spacy_install(
  version = "2.3.5", # since `spacy_install` accepts only this specification style.
  lang_models = "ja_core_news_sm",
  python_version = "3.9",
  pip = TRUE
)
```

This is because the style for specifying version is different between `conda install` and `pip install`, however, [spacy_pkgs](https://github.com/quanteda/spacyr/blob/06b4eadd421a0f94c0645dd4297c0e60eec8545d/R/spacy_install.R#L452-L458) always prepends an '=' to version number internally.

Is there any ways to install spacy v2.x using `spacy_install(pip = TRUE)`?

**Comment by kbenoit** [comment]
Would it not work to do as you have done above, skip the `spacy_install()` step, and then use:
```r
spacy_initialize(
  model = "ja_core_news_sm",
  virtualenv = "spacy+,
  condaenv = "spacy_condaenv", # created manually but also happens to be the default name
)
```

**Comment by kbenoit** [comment]
Solved in v1.3.
